@navikt/ds-css/baseline/utility.css

```diff
@@ -19,7 +19,6 @@
   position: absolute;
   white-space: nowrap;
   width: 1px;
-  /* 1 */
```

Comment:
Kommentaren `/* 1 */` blir pekt på av kommentaren på linje 7, så jeg stemmer for at vi beholder den.